DOCKER.md

```diff
@@ -50,6 +50,17 @@ will give you the most detailed.
 docker run -d --name s3server -p 8000:8000 -e LOG_LEVEL=trace scality/s3server
 ```
 
+#### SSL
+
+Set to true, this variable allows you to run S3 with SSL.
+
+```shell
+docker run -d --name s3server -p 8000:8000 -e SSL=true -e HOST_NAME=s3.scality.test
+scality/s3server
+```
```

Comment:
Maybe add a "When using, make sure to add s3.scality.test as an alias for localhost"? I know it prints at deployment, but repeating information is an easy way to avoid repeating questions in the FAQ.